streams/writable-streams/aborting.js

```diff
@@ -399,41 +402,35 @@ promise_test(t => {
 
 promise_test(() => {
   let resolveWrite;
-  let abortCalled = false;
-  const ws = new WritableStream({
+  const ws = recordingWritableStream({
     write() {
       return new Promise(resolve => {
         resolveWrite = resolve;
       });
-    },
-    abort() {
-      abortCalled = true;
     }
   });
 
   const writer = ws.getWriter();
   return writer.ready.then(() => {
     writer.write('a');
-    const abortPromise = writer.abort();
+    const abortPromise = writer.abort('b');
     return flushAsyncEvents().then(() => {
-      assert_false(abortCalled, 'abort should not be called while write is in-flight');
+      assert_array_equals(ws.events, ['write', 'a'], 'abort should not be called while write is pending');
```

Comment:
Ah, maybe the rewording has been overwritten when you merged the change from close-then-abort branch.